php_zookeeper.h

```diff
@@ -34,6 +34,18 @@ extern zend_module_entry zookeeper_module_entry;
 #define PHP_ZOOKEEPER_API
 #endif
 
+#ifndef TSRMLS_D
```

Comment:
It might be good to drop this compatibility support and remove the TSRMLS macros throughout the extension, since presumably this extension only supports PHP 7 and 8 now?

Also for when ZTS is enabled, there might need some additional changes...

Reply:
I'm of the same opinion regarding removing the TSRMLS macros, but I have so little familiarity with either the old or new thread safe local storage that I figured I'd let someone more knowledgeable make that call 😅